src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpSimplifyTypeNamesDiagnosticAnalyzer.cs

```diff
@@ -52,7 +52,9 @@ protected override void AnalyzeCodeBlock(CodeBlockAnalysisContext context)
 
             var syntaxTree = semanticModel.SyntaxTree;
             var options = context.Options;
+#pragma warning disable CS0612 // Type or member is obsolete
             var optionSet = options.GetDocumentOptionSetAsync(syntaxTree, cancellationToken).GetAwaiter().GetResult()!;
+#pragma warning restore CS0612 // Type or member is obsolete
```

Comment:
❔ Why are these not changing?

Reply:
Thank you for pointing this out. I mindlessly fixed it up when resolving merge conflicts.